.eslintignore

```diff
@@ -1,2 +1,3 @@
-node_modules
-lib
\ No newline at end of file
+/node_modules
+/lib
```

Comment:
Prepending slash means that the ignore will be applied only to the root of the repo.